src/com/firebirdberlin/nightdream/ui/ClockLayout.java

```diff
@@ -271,7 +273,10 @@ private void toggleDivider() {
             if (showDivider) {
                 divider.setVisibility(VISIBLE);
             }
-            setBackgroundColor(Color.parseColor("#44000000"));
+            GradientDrawable shape =  new GradientDrawable();
+            shape.setCornerRadius( Settings.getRoundCorners(context) );
```

Comment:
My suggestion is to remove the preference and to set the radius to 50
```suggestion
            shape.setCornerRadius(50);
```